docs/sphinx/source/reference/effects_on_pv_system_output/spectrum.rst

```diff
@@ -10,3 +10,5 @@ Spectrum
    spectrum.get_example_spectral_response
    spectrum.get_am15g
    spectrum.calc_spectral_mismatch_field
+   spectrum.first_solar_spectral_correction
```

Comment:
Should this be `spectrum.mismatch.first_solar_spectral_correction` ?  Or does the import in `spectrum.__init__` handle the path?

Reply:
> Or does the import in `spectrum.__init__` handle the path?

Yep.  I think sphinx's autodoc doesn't fetch docstrings by scanning `.py` files, but rather by importing the specified object and accessing its `__doc__` attribute.  So as long as the name is importable with the specified name it shouldn't matter where the original definition lives.  The choice here is whether we want `mismatch` to be included in the display text for the link.